src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -27,6 +28,11 @@ public override void Write(string message)
             {
                 _writer.Write(message);
             }
+
+            public override void WriteLine(string message)
+            {
+                _writer.WriteLine(message);
+            }
```

Comment:
I talked offline with @safern and we think it is better to have the same desktop behavior. so @safern is going to fix the library and then fix the test to test the expected behavior.